ansible/playbooks/roles/kubernetes_master/templates/kubeadm-config.yml.j2

```
@@ -61,3 +61,4 @@ kind: KubeletConfiguration
 apiVersion: kubelet.config.k8s.io/v1beta1
 cgroupDriver: systemd
 rotateCertificates: true
+enableControllerAttachDetach: {{ kubernetes_node_vars.specification.enable_controller_attach_detach }}
```

After moving parameter enable_controller_attach_detach to `schema/common/defaults/configuration/kubernetes-master.yml` it can look like that:
```suggestion
enableControllerAttachDetach: {{ specification.enable_controller_attach_detach }}
```

---

Done